src/citrine/jobs/waiting.py

```diff
@@ -130,13 +130,13 @@ def wait_while_validating(
 def wait_while_executing(
     *,
     collection: Union[
-        Collection[PredictorEvaluationExecution],
+        Collection[PredictorEvaluation],
```

Comment:
`PredictorEvaluation` should have been included previously. Same with the similar changes below.